root/etc/cont-init.d/30-config

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/with-contenv bash
+
+# permissions
+chown -R abc:abc \
+	/app \
```

Comment:
the `/app` permissions perhaps should not be changed. it will slow down the container boot process and make docker copy of all files there to accommodate permission changes.